pkg/component/kubernetes/apiserver/deployment.go

```diff
@@ -460,8 +475,8 @@ func (k *kubeAPIServer) computeKubeAPIServerArgs() []string {
 
 func (k *kubeAPIServer) etcdServersOverrides() string {
 	addGroupResourceIfNotPresent := func(groupResources []schema.GroupResource, groupResource schema.GroupResource) []schema.GroupResource {
-		for _, resource := range groupResources {
-			if resource.Group == groupResource.Group && resource.Resource == groupResource.Resource {
+		for _, res := range groupResources {
+			if res.Group == groupResource.Group && res.Resource == groupResource.Resource {
```

Comment:
Nit: Not sure I like the change from `resource` => `res` as I found it more readable before, but I see that we use the shorter variant also in some other places in the code.
WDYT?

(The same for 13 lines below).

Reply:
I mainly changed it because it shadows the `k8s.io/apimachinery/pkg/api/resource` import and my linter was complaining. I don't like ignoring linter complaints even if they are small b/c of potential linter fatigue (i.e. ignoring many linter warnings until you've got a tricky bug in the code that comes to haunt you later)

I can also revert this, it's just my personal preference.